atomic_reactor/plugins/post_pulp_sync.py

```diff
@@ -208,18 +208,22 @@ def run(self):
             if image.pulp_repo not in repos:
                 repo_id = self.create_repo_if_missing(pulp,
                                                       image.pulp_repo,
-                                                      image.repo)
+                                                      image.to_str(registry=False, tag=False))
```

Comment:
Turns out this case is checked in `test_create_missing_repo` so this change is not needed